main.js

```diff
@@ -0,0 +1,153 @@
+console.log(axios);
+
+const searchBar = document.getElementById("searchBar");
+const searchButton = document.getElementById('searchButton');
+let response;
+
+searchBar.addEventListener('keypress', async (e) => {
+    try {
+        if (e.key === 'Enter') {
```

Comment:
Het liefst wil je zo min mogelijk indenting. Dit kun je doen door de inverse te gebruiken van het statement, en zodra dat waar is te returnen:
```
if (e.key !== 'Enter') return;

blablabla hier komt code zonder extra indent
```